### PR TITLE
Fix tv.nate.com advertisement on AdGuard for iOS

### PR DESCRIPTION
**Pull request by piquark6046**
Fix https://github.com/List-KR/List-KR/issues/643

Reference: https://github.com/List-KR/List-KR/commit/f9b8cd180b3d72b0ea4d5a16cfd3ba6579d98339

### CODE REVIEW

File: filters/ThirdParty/filter_227_List-KR/template.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 !
-@include "https://raw.githubusercontent.com/List-KR/List-KR/master/filter-AdGuard.txt" /exclude="../../exclusions.txt"
\ No newline at end of file
+@include "https://raw.githubusercontent.com/List-KR/List-KR/master/filter-AdGuard.txt" /exclude="../../exclusions.txt"
```
Comment (Alex-302) [inline filters/ThirdParty/filter_227_List-KR/template.txt:2]: Why is it required? I see it exists in https://github.com/List-KR/List-KR/commit/f9b8cd180b3d72b0ea4d5a16cfd3ba6579d98339#diff-4cf7963c9d3cb7d9b7d2e8f7b0687f8cd5021e85d1900dd69741e8f3bc9efc29

Comment (piquark6046) [reply filters/ThirdParty/filter_227_List-KR/template.txt:2]: `trustLevel` of List-KR is `high` disallowing any javascript rule.

Comment (Alex-302) [reply filters/ThirdParty/filter_227_List-KR/template.txt:2]: Better discuss in our chat.

**Review by Alex-302 — APPROVED** [review]

**Comment by piquark6046** [comment]
```javascript
var BuildObs, PlayerObs
let PlayerCall = (mutationList, observer) => {
    mutationList.forEach((e) => {
        if (Array.from(e.target.classList).some((e) => {
                return e === "ntv-ad-playing"
            })) {
            player.trigger("adended");
            PlayerObs.disconnect()
        }
    })
}
let BuildCall = (mutationList, observer) => {
    mutationList.forEach((e) => {
        Array.from(e.addedNodes).filter((o) => {
            return o.id === "natePlayer"
        }).forEach((k) => {
            PlayerObs = new MutationObserver(PlayerCall);
            PlayerObs.observe(k, {
                attributes: true
            });
            BuildObs.disconnect()
        })
    })
}
BuildObs = new MutationObserver(BuildCall)
BuildObs.observe(document.querySelector(".player-wrapper"), {
    childList: true
})
```

**Review by zloyden — APPROVED** [review]

**Pull request merged**